js/src/setup-mc/setup-stepper/setup-paid-ads/paid-ads-setup-sections.js

```diff
@@ -45,23 +43,8 @@ const defaultPaidAds = {
  * @param {Array<CountryCode>} targetAudience Country codes of selected target audience.
  * @return {PaidAdsData} The resolved paid ads data.
  */
-function resolveInitialPaidAds( paidAds, targetAudience ) {
-	const nextPaidAds = { ...paidAds };
-
-	if ( targetAudience ) {
-		if ( nextPaidAds.countryCodes === defaultPaidAds.countryCodes ) {
-			// Replace the country codes with the loaded target audience only if the reference is
-			// the same as the default because the given country codes might be the restored ones.
-			nextPaidAds.countryCodes = targetAudience;
-		} else {
-			// The selected target audience may be changed back and forth during the onboarding flow.
-			// Remove countries if any don't exist in the latest state.
-			nextPaidAds.countryCodes = nextPaidAds.countryCodes.filter(
-				( code ) => targetAudience.includes( code )
-			);
-		}
-	}
-
+function resolveInitialPaidAds( paidAds, targetAudience = [] ) {
```

Comment:
By setting an empty array as the default parameter, it seems targetAudience isn’t required, and we can create ads without specifying any countries.

Reply:
Thanks @jorgemd24 . Happy to revert that change but when I tested the "Continue" button to proceed to the next step was disabled if there were no countries selected. Screenshot below:
![image](https://github.com/user-attachments/assets/3d3381c4-e54d-4cc4-83b1-1069deed6535)

Also, in the `validateCampaign` function, there's a check for an empty array of countries.
https://github.com/woocommerce/google-listings-and-ads/blob/93b4180556cd6eeba69906f7e9bab8c8880b1c4e/js/src/components/paid-ads/validateCampaign.js#L19

Let me know what you think 😁